### ISSUE TITLE
Learning rate & schedule

### ISSUE THREAD

**Issue by malteos** [Muennighoff/sgpt#7]
Hi @Muennighoff 

thanks for your interesting paper and for sharing this repo! You seem using different learning rates and different schedules (sometimes [with warm-up](https://github.com/Muennighoff/sgpt/blob/77ed39c6073e1d5a608814a4b1048251f3becb37/biencoder/nli_msmarco/sentence-transformers/examples/training/ms_marco/train_bi-encoder_mnrl.py#L56-L57), [sometimes constant LR](https://github.com/Muennighoff/sgpt/blob/40350ef7083fa5c79ea334b4b5c28caf748dafa8/biencoder/nli_msmarco/sentence-transformers/examples/unsupervised_learning/TSDAE/train_askubuntu_tsdae.py#L102-L103)).

Did you find these settings empirically? Do they depend on the model size or dataset? Is there any particular LR setting you would recommend when training a GPT model with BitFit?

**Comment by Muennighoff** [comment]
Hey, thanks for reaching out - Here are some thoughts on LR & Scheduling:
1) LR should be [higher for BitFit than non-BitFit](https://arxiv.org/abs/2106.10199)
2) I tend to[ increase LR when increasing batch size](https://miguel-data-sc.github.io/2017-11-05-first/#:~:text=For%20the%20ones%20unaware%2C%20general,descent%20(batch%20size%201)
3) Decaying towards the end may help, but BitFit tends to be less prone to overfit anyways as you don't finetune a lot of parameters

**Comment by malteos** [comment]
Thanks for the quick reply.
